_posts/2024-04-15-gsi-tech-blog.md

```diff
@@ -0,0 +1,75 @@
+
+The APU technology family is a Compute-in-memory (CiM) architecture seeking to eliminate computation time wasted on data movement by closing the traditional gap between processors (millions in GSI's case) and storage elements, which is highly beneficial for data-intensive workloads. Computation occurs where the data lives - in the storage elements, avoiding traditional Processor/Memory bottlenecks!
+
+## Several applications
+
+Many institutions often use things like SAR (synthetic aperture radar) to monitor landscapes to check for things like the progress of deforestation, forest fires, as well as the erosion of coastal areas. Even corporations use this tech to count the number of cars in their stores’ parking lots to help forecast business decisions. We can even use this tech to predict and detect ships adrift at sea for the purposes of rescue. The same advantage is available here. Instead of sending entire collections of satellite data, we now have the potential to send results that represent those changes instead.
+
+If you’re curious about what GSI Technology works on you should definitely head over to their [website](https://gsitechnology.com). They’re applying AI and ML to a variety of modern problems. There are many potential applications that are up to their imagination - aerospace, genomics, computer vision, cyber security for space, vector search in space, data centers in space for the purposes of computing large volumes of data and more.
+
+## Parting food for thought
+
+Satellites and space sure sound cool, but there may be many other places in which the observability of our own Earth can be improved. Not only on a cosmic scale, but any place in which we collect large amounts of data. Imagine a microscope with a bit of computational memory, equivalent to RAM and a CPU that could supplement the years and years of experience doctors and scientists have accumulated. Not replace, but supplement. Imagine an MRI machine equipped with a model to recognize brain scan anomalies. As a long time sufferer of epilepsy, a condition for which only three out of ten cases ever have a root cause determined, I sure wouldn’t mind donating the vector of my brain scans to science. With an open source license, of course.
```

Comment:
```suggestion
Satellites and space sure sound cool, but there may be many other places in which the observability of our own planet can be improved. This may be true not only on a cosmic scale but for any application in which you collect large amounts of data. Imagine a microscope with a bit of computational memory, equivalent to RAM and a CPU, that could supplement the years and years of experience that doctors and scientists have accumulated (not replace, but supplement). Imagine an MRI machine equipped with a model that recognizes brain scan anomalies. As a longtime sufferer of epilepsy, a condition for which only 3 out of 10 cases ever have a root cause determined, I sure wouldn’t mind donating the vector of my brain scans to science---with an open-source license, of course.
```